primitives/npos-elections/compact/src/lib.rs

```diff
@@ -169,7 +169,7 @@ fn struct_def(
 		);
 		quote!{
 			#compact_impl
-			#[derive(Default, PartialEq, Eq, Clone, Debug)]
+			#[derive(Default, PartialEq, Eq, Clone, Debug, PartialOrd, Ord)]
```

Comment:
why do we need to derive an order here ?
I slightly worry that this order is confused with the solution score, but it should be fine

also if we derive it for the compact encoding I don't see why we shouldn't derive it for the not compact one just below

Reply:
The `Ord` implementations here and later are dead code now, technically. They were needed for an earlier approach which stored the solution set in a `StorageValue<BTreeSet>`. I can take them out; I just figured that they should be harmless to leave.

Reply:
ok then I don't mind